churchland_pipeline/processing.py

```diff
@@ -11,17 +11,17 @@
 class EmgSpikeSorter(dj.Lookup):
     definition = """
     # Spike sorter for EMG data
-    emg_sorter_name : varchar(64)
-    emg_sorter_version : varchar(8)
+    emg_sorter_name : varchar(255)
+    emg_sorter_version : varchar(255)
     ---
     """
     
 @schema    
 class NeuralSpikeSorter(dj.Lookup):
     definition = """
     # Spike sorter for neural data
-    neural_sorter_name : varchar(64)
-    neural_sorter_version : varchar(8)
+    neural_sorter_name : varchar(255)
+    neural_sorter_version : varchar(255)
```

Comment:
try to keep primary key attributes small. MySQL limits the total length of the primary key.  Also, since primary key attributes become incorporated into other tables through dependencies, there is duplication of storage. Select the shortest identifiying attributes.